electron/output/Forsta Messenger/pre-appx/app/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2013-2018 GitHub Inc.
```

Comment:
copyright github?

Reply:
Weird.  I'll dig into where that came from.